webroot/adm/oncall/generate-api-key-secret.html

```diff
@@ -0,0 +1,641 @@
+<html>
+
+<head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+    <script src="/js/main.js"></script>
+    <link href="/css/bootstrap.min.css" rel="stylesheet">
+    <link href="/css/style.css" rel="stylesheet">
+    <script src="/js/bootstrap.bundle.min.js"></script>
+</head>
+
+<body>
+    <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+        <symbol id="exclamation-triangle-fill" fill="currentColor" viewBox="0 0 16 16">
+            <path
+                d="M8.982 1.566a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767L8.982 1.566zM8 5c.535 0 .954.462.9.995l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995A.905.905 0 0 1 8 5zm.002 6a1 1 0 1 1 0 2 1 1 0 0 1 0-2z" />
+        </symbol>
+    </svg>
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" style="display: none;" fill="currentColor"
+        class="bi bi-copy" viewBox="0 0 16 16">
+        <path fill-rule="evenodd"
+            d="M4 2a2 2 0 0 1 2-2h8a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2zm2-1a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1zM2 5a1 1 0 0 0-1 1v8a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1v-1h1v1a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V6a2 2 0 0 1 2-2h1v1z" />
+    </svg>
+
+    <div class="container-lg">
+        <div class="col-lg">
+            <div class="row">
+                <h1>UID2 Env - On-call - Generate Client API Key/Secret</h1>
+            </div>
+            <div class="row pt-3 pb-2">
+                <div class="col-sm">
+                    <a href="/">Back</a>
+                </div>
+            </div>
+            <div class="row py-3">
+                <div class="card card-body">
+                    <div class="row py-1">
+                        <div class="col">
+                            <h4>Instructions</h4>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col">
+                            <ol>
+                                <li>First seach for a participant with the requested name from the Jira ticket.</li>
```

Comment:
seach -> search

Reply:
We also don't need "First", it's already the first list item:
"Search for a participant with the requested name from the Jira ticket."